From ffb405fd1c24d303104b76314a56cab65b4971a2 Mon Sep 17 00:00:00 2001
From: Radha Mohan Chintakuntla <rchintakuntla@cavium.com>
Date: Tue, 11 Dec 2018 22:23:46 +0300
Subject: [PATCH 174/767] misc: otx_bphy_ctr: Allow maximum interrupts of 27 to
 userspace in BPHY

commit 1096f59baf4310f7e76277a8162e6c830d80a481 from
git@git.assembla.com:cavium/WindRiver.linux.git

Change-Id: I51845d70c3ad4a44fe03367ac66aef2e74e3df71
Signed-off-by: Radha Mohan Chintakuntla <rchintakuntla@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/1777
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/8030
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/misc/otx_bphy_ctr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/otx_bphy_ctr.c b/drivers/misc/otx_bphy_ctr.c
index 05087050a2e6..7818ca69d345 100644
--- a/drivers/misc/otx_bphy_ctr.c
+++ b/drivers/misc/otx_bphy_ctr.c
@@ -20,7 +20,7 @@
 
 #define DEVICE_NAME	"otx-bphy-ctr"
 #define OTX_IOC_MAGIC	0xF3
-#define MAX_IRQ		23
+#define MAX_IRQ		27
 
 static struct device *otx_device;
 static struct class *otx_class;
-- 
2.31.1

